spec.bs

```diff
@@ -144,6 +144,9 @@ Private State Token operations rely on [[FETCH]]. A fetch request corresponding
 specific Private State Token operation can be created and used as a parameter to the
 fetch function.
 
+Note that this specification uses the terms "masked" and "unmasked" where
```

Comment:
Can we use the `Note:` syntax? https://speced.github.io/bikeshed/#notes-etc that'll make it stick out.

Reply:
(so just "Note: This spec...")

Reply:
Done